bluetooth/src/androidLibMain/kotlin/device/DefaultDeviceConnectionManager.kt

```diff
@@ -110,11 +106,7 @@ internal actual class DefaultDeviceConnectionManager(
 
         override fun onDescriptorWrite(gatt: BluetoothGatt?, descriptor: BluetoothGattDescriptor?, status: Int) {
             descriptor ?: return
-            // TODO update implementation so it doesn't depend on descriptor.value which is deprecated
-            // https://github.com/splendo/kaluga/issues/609
-            // https://developer.android.com/reference/android/bluetooth/BluetoothGattDescriptor#getValue()
-            @Suppress("DEPRECATION")
-            updateDescriptor(descriptor, descriptor.value, status)
+            handleUpdatedDescriptor(descriptor.uuid, status == GATT_SUCCESS)
```

Comment:
For the sake of consistency with code of above, `succeeded = status == GATT_SUCCESS`